crates/bevy_app/src/main_schedule.rs

```diff
@@ -74,6 +74,10 @@ pub struct RunFixedUpdateLoop;
 ///
 /// Frequency of execution is configured by inserting `Time<Fixed>` resource, 64 Hz by default.
 /// See [this example](https://github.com/bevyengine/bevy/blob/latest/examples/time/time.rs).
+///
+/// Note that schedules are not run in parallel, in particular,
+/// systems in the `FixedUpdate` schedule are not run in parallel with systems
```

Comment:
"systems in the `FixedUpdate` schedule are all run before systems in the `Update` schedule"

next question people are going to ask is: "okay, what's the order of execution then?", and you can change just a single word in docs to immediately answer it